tests/testthat/_snaps/ard_categorical.md

```diff
@@ -22,12 +22,32 @@
       variable, variable_level, stat_name, statistic, statistic_fmt))
     Output
         variable variable_level stat_name statistic statistic_fmt
-      1       am              0         n        19            19
+      1       am              0         n        19          19.0
```

Comment:
These snapshots should not update

Reply:
So, this happened because `n` is not covered in the custom formatting supplied for `am`, so it falls back on the global default of rounding to 1 decimal place. 

Would you expect that it combines user-supplied with the default_fmt_fn() in this case? Maybe an update to `fill_formula_selectors`?

Reply:
Hmmm, the more i think about this, maybe we should just have a few defaults (e.g. default for percentages and integers), and everything else is rounded to one decimal place.

Then we can leave the default value of `fmt_fn = NULL`, and just leave it at that. What do you think?